### ISSUE TITLE
When using the When-rule the error message is '1' when no else is provided

### ISSUE THREAD

**Issue by jonsa** [Respect/Validation#739]
``` php
$v = new \Respect\Validation\Rules\When(
    new \Respect\Validation\Rules\AlwaysInvalid,
    new \Respect\Validation\Rules\AlwaysValid
);

try {
    $v->assert('foo');
} catch (\Respect\Validation\Exceptions\ValidationException $e) {
    echo $e->getMainMessage();
}
```

prints `1`

**Comment by danimalweb** [comment]
I have the same issue. Did you find a solution?

**Comment by henriquemoody** [comment]
Fixed on [1.0.20](https://github.com/Respect/Validation/releases/tag/1.0.20) and [1.1.15](https://github.com/Respect/Validation/releases/tag/1.1.15).

Thanks for reporting! :panda_face:
